slidechain/slidechain_test.go

```diff
@@ -266,8 +266,16 @@ func TestImport(t *testing.T) {
 }
 
 func TestEndToEnd(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
```

Comment:
Just curious — is 1 minute no longer enough time to run this test?

Reply:
The test for a single input case takes ~40 seconds, so no. However, the 3 is a vestige from when the goroutines were being launched anew for each test case. Now, the end-to-end takes ~90s consistently, so I'm happy to just make this 2 minutes.

Reply:
Ah, yes! Adding more test cases answers my question :) I think keeping it at 3 is probably still a totally reasonable limit